lib/tasks/openapi.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'open_api'
+
+desc 'Generate openapi V3 documentation'
+namespace :documentation do
+  task :generate => :environment do
```

Comment:
`SimpleCov` needs to be started before application code is loaded. This effectively breaks code coverage collection for tests (specs still work).